retaildevcrews/relayrunner
# Add Change Feed Process for clientStatus

# Type of PR

- [x] Test changes
- [ ] GitHub Template changes

## PR Checklist



- [x] I ran lint checks locally prior to submission.

## Purpose of PR
Write to console when Cosmos DB change feed processor starts and item change handler executes

## Does this introduce a breaking change



## Validation

- Run backend application
- Verify logs that relay runner is running
- Update item in CosmosDB clientStatus
- Verify logs that processor executed
- Create new item in CosmosDB clientStatus
- Verify logs that processor executed

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/239
